packages/shared/src/hooks/useCoresFeature.ts

```diff
@@ -35,6 +37,24 @@ export const useCanAwardUser = (
   return hasAccess && !isSpecialUser && canAwardUser(props);
 };
 
+export const useCanAwardSquad = (props): typeof LoggedUser => {
```

Comment:
shoud it not just return `boolean`?

Reply:
It could (which I had) but we need the first eligible admin to pass to the modal.
So decided to do it in one action.

Can split it if you prefer, but somewhere we need to get that first admin again.

Reply:
ah get it, maybe then call it `useSquadAwardAdmin`, or splitting is also cool 👍